.gitignore

```diff
@@ -15,6 +15,8 @@ TAGS
 .idea
 .idea/*
 .idea_modules
+.run/
```

Comment:
This is not related to BSP nor Bloop, but it's useful to keep `Run configurations` separate from the project files stored in `.idea` directory